functests/1_performance/performance.go

```diff
@@ -781,6 +782,19 @@ func validatTunedActiveProfile(nodes []corev1.Node) {
 	var err error
 	var out []byte
 	activeProfileName := components.GetComponentName(testutils.PerformanceProfileName, components.ProfileNamePerformance)
+
+	// check if some another Tuned profile overwrites PAO profile
+	tunedList := &tunedv1.TunedList{}
+	if err := testclient.Client.List(context.TODO(), tunedList); err != nil {
+		klog.Error(err)
+	}
+	for _, t := range tunedList.Items {
+		if strings.Contains(*t.Spec.Profile[0].Data, fmt.Sprintf("include=%s", activeProfileName)) {
+			klog.Warning(fmt.Sprintf("PAO tuned profile amended by '%s' profile, test may fail", t.Name))
+			activeProfileName = *t.Spec.Profile[0].Name
```

Comment:
the same check that `t.Spec.Profile[0].Name != nil` before dereferece

Reply:
hm, not sure if this object can be without _Name_ field, but added checking that.. thanks